graphprot/NeuralNet.py

```diff
@@ -346,10 +346,10 @@ def test(self, database_test, threshold=4, hdf5='test_data.hdf5'):
         
         # If file exists, change its name with a number 
         count = 0
+        hdf5_name = hdf5.split('.')[0]
         while os.path.exists(fname) : 
             count += 1
-            hdf5 = hdf5.split('.')[0]
-            hdf5 = '{}_{:03d}.hdf5'.format(hdf5, count)
+            hdf5 = '{}_{:03d}.hdf5'.format(hdf5_name, count)
             fname = os.path.join(self.outdir, hdf5)
```

Comment:
you could consider putting that in a staticmethod or external function as it appears at several places :)

Reply:
in general code duplication should be avoided as it makes the code difficult to maintain. In this case it's not a big deal of course :)